demo/index.js

```diff
@@ -21,6 +22,9 @@ if (!isBenchmark) {
 	initDevTools();
 }
 
+// mobx-state-tree fix
+window.setImmediate = setTimeout;
```

Comment:
@mweststrate Are there any particular reasons to use `setImmediate` over `setTimeout`?

Reply:
Also interested to hear why this was used. Looks like it's here:
https://github.com/mobxjs/mobx-state-tree/blob/d3e24ab5f89e0e197e2b8594a6de774f3e739351/packages/mobx-state-tree/src/core/flow.ts#L117